lib/internal/checkUrl.js

```diff
@@ -24,6 +24,25 @@ function checkUrl(link, baseUrl, options, callback)
 		callHandler.async(callback, link);
 		return;
 	}
+
+	if (cache != null)
```

Comment:
i'm retarded today.. gonna have a coffee. Both are the same

# ===
> 1. If Type(x) is different from Type(y), return false.
2. If Type(x) is Undefined, return true.
3. If Type(x) is Null, return true.

# ==
> 1. If Type(x) is the same as Type(y), then  
a. If Type(x) is Undefined, return true.  
b. If Type(x) is Null, return true.